luda-editor/client/src/pages/project_list_page/mod.rs

```diff
@@ -141,9 +140,7 @@ impl ProjectListPage {
             [MouseButton::Left],
             move |event| {
                 if event.button == Some(MouseButton::Left) {
-                    namui::event::send(router::Event::Route(Arc::new(move || {
-                        router::Route::SequenceListPage(SequenceListPage::new(project_id))
-                    })));
+                    Router::move_to(format!("/sequence_list/{project_id}"));
```

Comment:
Could you let user don't make mistake about url? i hope user can route using enum, not string directly.

Reply:
sure